src/components/update-service/update-service.controller.ts

```diff
@@ -213,74 +213,28 @@ export class UpdateServiceController implements angular.IController {
     this.ctrl.orderComplete = false;
     this.ctrl.error = null;
 
-    var parameterData: any = this.getParameters(this.ctrl.parameterData);
+    let parameterData: any = this.getParameters(this.ctrl.parameterData);
```

Comment:
If we're going to switch, we should use `const` here and below :)

Reply:
agree, i can change this.